app/decorators/guide_decorator.rb

```diff
@@ -1,6 +1,19 @@
-class GuideDecorator < Draper::Decorator
-  delegate :id, :slug, :concise_label, :description,
-           :option?, :related_to_appointments?, :related_to_booking?
+class GuideDecorator
+  delegate(
+    :id,
+    :slug,
+    :concise_label,
+    :description,
+    :option?,
+    :related_to_appointments?,
+    :related_to_booking?,
+    :==,
+    to: :guide
+  )
```

Comment:
Would subclassing `SimpleDelegator` be more concise than delegating all these methods?